### DISCUSSION TITLE
Your Youtube links are broken

### DISCUSSION THREAD

**Discussion by juliagrabowski** [Riverscapes/gcd#413] [General]
Hi! I've been following along with your Youtube video tutorials for a while, and noticed that in the last few days the links to the videos broke. Just a heads up!

**Comment by philipbaileynar** [comment]
@joewheaton I think this might be a change by YouTube. You might need to make the reset the status of very old videos unlisted and copy new links.

**Comment by joewheaton** [comment]
Thanks @juliagrabowski for pointing this out. I just had a look through a bunch of different GCD pages with embedded videos and they seem to be working for me. But I think that is just because YouTube recognizes me logged in on my browser. However, I looked around in my Videos on YouTube and saw many of the really old ones were all changed to Private instead of unlisted @philipbaileynar. I  am updating their status now, so hopefully that does the trick.  I will try and get them all (there are 100s)
